Formula/warp.rb

```diff
@@ -0,0 +1,24 @@
+class Warp < Formula
+  desc "Secure terminal sharing with one simple command"
+  homepage "https://github.com/spolu/warp"
+  url "https://github.com/spolu/warp.git",
+    :tag => "v0.0.3",
+    :revision => "8a04cd7c95973e168a9bf2d2db6522574302164d"
```

Comment:
We prefer a `.tar.gz` (e.g. https://github.com/spolu/warp/archive/v0.0.3.tar.gz) to a git tag